packages/extra-shot-mocha/package.json

```diff
@@ -38,6 +38,7 @@
     "eslint-plugin-import": "^2.25.2",
     "eslint-plugin-no-secrets": "^0.8.9",
     "eslint-plugin-prettier": "^4.0.0",
+    "@microsoft/eslint-plugin-teamsfx": "file:../eslint-plugin-teamsfx",
```

Comment:
Why not move eslint-plugin-teamsfx to extra-shot-mocha?
Or instead of version

Reply:
peer deps issue